src/lib/web-component/filter-option/filter-option-element.js

```diff
@@ -6,8 +6,8 @@ export class FilterOption extends HTMLElement {
     super();
 
     this.viewCheckboxLabels = new Set();
-    // It contains the checked checkboxes
-    this.activeCheckboxLabels = new Set();
+    // It contains the unchecked checkboxes
+    this.inactive = new Set();
```

Comment:
This name is too vague. If you choose a meaningful name, the comment wouldn't be needed.
How about `uncheckedCheckboxLabels`?

Reply:
Yeah, `uncheckedCheckboxLabels` sounds great. I also removed the comment.